### DISCUSSION TITLE
Looking for help w this codebase

### DISCUSSION THREAD

**Discussion by aryzach** [fireact-dev/main#19] [General]
Hi, I'm hoping to find somebody familiar with this code base that could walk me though the setup. I've followed the installation, but am running into walls. I have a little React experience, but none with full-stack integration. I've setup Firebase with Stripe extension, and the Firestore. I don't believe I'll need Firebase / Cloud Functions or Mailgun. Thank you

**Comment by lilbro1062000** [comment]
you will need the functions to  create accounts so you need the functions
deployed.

**Comment by chaoming** [comment]
All the steps are required except the step to setup tax rate. As the project is not a pure front-end project, it also handles subscription payments, it will require the Firebase, Cloud Functions and Mailgun to work.
